.github/workflows/support-workers-build.yml

```diff
@@ -83,8 +83,11 @@ jobs:
                 app: workers
                 parameters:
                   templatePath: cfn.yaml
+                dependencies: [support-workers]
               support-workers:
                 type: aws-lambda
+                actions:
+                  - uploadLambda
```

Comment:
we only need to upload the jar file now, and the cfn will change every time ( 😢 slow) so it will delete the old version and create a new one rather than calling update-function which only updates $LATEST.
Perhaps there's scope for an improvement to riffraff to handle that rather than hacking it with CFN?